weave-js/src/components/PagePanelComponents/Home/HomeFeaturedTemplates.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import * as LayoutElements from './LayoutElements';
+import {Button} from '../../Button';
+import {Type, callOpVeryUnsafe} from '@wandb/weave/core';
+import {useNodeValue} from '@wandb/weave/react';
+import * as Tabs from '@wandb/weave/components/Tabs';
+
+import {HomePreviewSidebarTemplate} from './HomePreviewSidebar';
+import {SetPreviewNodeType} from './common';
+import styled from 'styled-components';
+
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import './github-markdown-light.css';
+import {TargetBlank} from '@wandb/weave/common/util/links';
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter';
+// import {dark} from 'react-syntax-highlighter/dist/esm/styles/prism';
+
+const TWDrawerHack = styled.div`
+  height: 100%;
+  overflow: hidden;
+  .tw-style {
+    height: 100%;
+  }
+`;
+
+type TemplateType = {
+  config_type: Type | null;
+  description: string;
+  display_name: string;
+  op_name: string;
+  instructions_md: string;
+};
+
+export const HomeFeaturedTemplates: React.FC<{
+  setPreviewNode: SetPreviewNodeType;
+}> = ({setPreviewNode}) => {
+  const featuredTemplatesNode = callOpVeryUnsafe(
+    'py_board-get_featured_board_templates',
+    {}
+  );
+  const featuredTemplates = useNodeValue(featuredTemplatesNode as any);
```

Comment:
I don't love how many `any`'s there are in weave code

Reply:
fixed in this case. Agreed.